gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -73,7 +73,7 @@
   // more complex implementation that periodically queries the service to find out the last
   // committed offset, to determine what's safe to discard, but that would also impose a performance
   // penalty.
-  private static final int NUMBER_OF_REQUESTS_TO_RETAIN = 5;
+  private static final int NUMBER_OF_REQUESTS_TO_RETAIN = 20;
```

Comment:
This should be configurable via property

Reply:
Done